go.mod

```diff
@@ -16,13 +16,13 @@ require (
 	github.com/kr/pretty v0.2.2-0.20200810074440-814ac30b4b18 // indirect
 	github.com/mvo5/goconfigparser v0.0.0-20200803085309-72e476556adb
 	// if below two libseccomp-golang lines are updated, one must also update packaging/ubuntu-14.04/rules
-	github.com/mvo5/libseccomp-golang v0.9.1-0.20180308152521-f4de83b52afb // old trusty builds only
+	github.com/mvo5/libseccomp-golang v0.9.1-0.20180308152521-f4de83b52afb // indirect; old trusty builds only
```

Comment:
Is change to this file intended?

Reply:
That was not intended, not sure what happend here. Sorry in any case.